automation/patch_igz/patch_remote.py

```diff
@@ -34,97 +33,122 @@
-            self._replace_deploy_policy()
-            self._replace_deployment_images(self.Consts.api_container, built_api_image)
-            if self._patch_log_collector:
-                # sanity
-                if not built_log_collector_image:
-                    raise RuntimeError("Log collector image not built")
+        if self._patch_log_collector_image:
+            self._replace_deployment_images(
+                Constants.log_collector_container,
+                target_to_built_images[Constants.log_collector],
+            )
+        if not self._skip_patch_api:
+            try:
+                # Replace deployment policies and images
+                self._patch_deployment_from_file()
                 self._replace_deployment_images(
-                    self.Consts.log_collector_container, built_log_collector_image
+                    Constants.api_container, target_to_built_images[Constants.api]
                 )
 
-            if self._reset_db:
-                self._reset_mlrun_db()
-            else:
-                self._rollout_deployment()
+                # Reset or rollout deployment as necessary
+                if self._reset_db:
+                    self._reset_mlrun_db()
+                else:
+                    self._rollout_deployment()
 
-            self._wait_deployment_ready()
-
-        finally:
-            out = self._exec_remote(
-                [
-                    "kubectl",
-                    "-n",
-                    "default-tenant",
-                    "get",
-                    "pods",
-                ],
-            )
+                self._wait_deployment_ready()
 
-            for line in out.splitlines():
-                if self.Consts.api_container in line:
-                    logger.info(line)
-                elif self.Consts.log_collector_container in line:
-                    logger.info(line)
+            finally:
+                # Check status of pods after deployment
+                out = self._exec_remote(
+                    ["kubectl", "-n", "default-tenant", "get", "pods"]
+                )
+                for line in out.splitlines():
+                    if (
+                        Constants.api_container in line
+                        or Constants.log_collector_container in line
+                    ):
+                        logger.info(line)
 
-            self._disconnect_from_node()
+                self._disconnect_from_node()
 
         logger.info(
-            "Deployed branch successfully! Yay! (Note this may not survive system restarts)"
+            "Deployed branch successfully! (Note: This may not survive system restarts)"
         )
 
     def _docker_login_if_configured(self):
         registry_username = self._config.get("REGISTRY_USERNAME")
         registry_password = self._config.get("REGISTRY_PASSWORD")
-        registry_url = self._config.get("REGISTRY_URL")
+        docker_registry = self._config.get("DOCKER_REGISTRY")
```

Comment:
need to remove `REGISTRY_URL` from `automation/patch_igz/patch_env_template.yml`